app/views/shared/_post.html.erb

```diff
@@ -0,0 +1,7 @@
+<div id="post" class="border-solid border-4 border-black md:w-1/2 mx-auto w-full mb-5">
+  <%= link_to user_posts_path(post) do %>
+    <h2 class="font-bold ml-3"><%= post.title %></h2>
+  <% end %>
```

Comment:
- [ ] @rmiyoyo Great work implementing the changes requested, however, you added the wrong link, the link should lead to the post-show page.